README.md

```diff
@@ -1,9 +1,13 @@
-# SwiftCache
+# swift-resource-provider
 
-A modular cache or data pipeline system.
+A modular resource fetching and management system.
 
-The API is loosely based on Combine/SwiftUI. It doesn't resolve caching logic complexity but helps organize it in a far
-more modular and testable way.
+This is like Combine, but for getting things instead of receiving them. It makes for an easy to understand, common
+abstraction of getting something repeatably based on a series of unique characteristics, as well as enabling more
+sophisticated workflows including but not limited to caching steps.
+
+As with many similar frameworks and language facilities, this doesn't make these complicated issues simple but it ought
+to help organize them in a far more modular and testable way.
```

Comment:
May want to add a link to auto-generated docs:
```
- [Documentation](https://swiftpackageindex.com/Gabardone/swift-resource-provider/documentation)
```

(should go live as soon as you rename and submit to swift package index)

Reply:
Oh that's going to be easier than generating them myself…